sky/jobs/controller.py

```diff
@@ -35,6 +36,10 @@
 # to inherit the setup from the `sky` logger.
 logger = sky_logging.init_logger('sky.jobs.controller')
 
+# Since sky.launch is very resource demanding, we limit the number of
+# concurrent sky.launch process to avoid overloading the machine.
+_MAX_NUM_LAUNCH = psutil.cpu_count() * 2
```

Comment:
Why do we use CPU as the threshold, while having calculation of memory threshold? Should we just use memory as threshold?